misc/security-and-privacy.md

```diff
@@ -0,0 +1,96 @@
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+This questionnaire has [moved](https://w3ctag.github.io/security-questionnaire/).
```

Comment:
What does "move" mean? From where to where?

Reply:
Oh I don't know, that's in the template I copied!

Reply:
Removed in e971621.